### DISCUSSION TITLE
Compressing logs

### DISCUSSION THREAD

**Discussion by podikoglou** [biomejs/biome#2177] [Ideas]
I just started using biome yesterday for my project and used it for a few hours, and I just noticed that it already has logs which total to 1.6MB
```
alex@hbo:~/.cache/biome/biome-logs $ du -sh
1.6M    .
```
I can only guess that they add up over time, so perhaps they could be compressed? Just an idea

**Comment by arendjr** [comment]
Thanks, we have an issue for this indeed: https://github.com/biomejs/biome/issues/1613

**Reply by podikoglou** [reply]
Oops, sorry for opening a duplicate, I didn't see the issue

**Reply by arendjr** [reply]
No problem, thanks for reporting!
